op-program/client/l2/hints.go

```diff
@@ -53,3 +57,19 @@ var _ preimage.Hint = L2OutputHint{}
 func (l L2OutputHint) Hint() string {
 	return HintL2Output + " " + (common.Hash)(l).String()
 }
+
+type L2BlockDataHint struct {
+	AgreedBlockHash common.Hash
+	BlockHash       common.Hash
+	ChainID         uint64
+}
+
+var _ preimage.Hint = L2BlockDataHint{}
+
+func (l L2BlockDataHint) Hint() string {
+	hintBytes := make([]byte, 32+32+32)
```

Comment:
Isn't a uint64 8 bytes so this should be 32+32+8? The extra padding won't hurt if we're consistent on both sides, but it's not immediately obvious that's happening from this code and would be easy to miss.

Reply:
This was done to allow for uint256 chainID in the future. Though I agree it's confusing right now, will stick with 8-bytes for now.